util/llpcInternal.cpp

```diff
@@ -749,4 +749,30 @@ bool IsIsaText(
     return (dataSize != 0) && ((reinterpret_cast<const char*>(pData))[0] == '\t');
 }
 
+// =====================================================================================================================
+// Manually add a target-aware TLI pass, so middle-end optimizations do not think that we have library functions.
+void AddTargetLibInfo(
+    legacy::PassManager*  pPassMgr,   // [in/out] Pass manager
+    Context*              pContext)   // [in] LLPC context
```

Comment:
Could you move up pContext as the first parameter, following this order

pContext,
pPassMgr

We often follow this order: input parameters, inout parameters, output parameters. Parameter such as pContext, pModule, pEntryPoint are often placed as the first parameter.